src/CommonLib/LDAPUtils.cs

```diff
@@ -84,6 +132,17 @@ public string ConvertWellKnownPrincipal(string sid, string domain)
             return $"{forest}-{sid}".ToUpper();
         }
 
+        private async Task<Group> GetBaseEnterpriseDC()
+        {
+            var forest = GetForest()?.Name;
+            if (forest == null) Logging.Debug("Error getting forest, ENTDC sid is likely incorrect");
+            var g = new Group {ObjectIdentifier = $"{forest}-S-1-5-9".ToUpper()};
```

Comment:
In the case where `forest` is `null` the OID will be `-S-1-5-9`. Is that what we want or do we want to deliberately prefix with some string?